src/main.js

```diff
@@ -67,7 +67,7 @@ async function findChangedBundles (bundles) {
         for (let filename of cached.includedFiles) {
           if (fasterHasFileChanged(filename)) {
             thisVariantHasChanged = true
-            break
+            // Don't break to ensure that we refresh the cached hash for any changed files
```

Comment:
i've tried to read through the flow here to see how breaking would've resulted in any different outcome than not breaking, but i couldn't.. can you explain your thoughts?

how i'm reading it: the for loop's outcome is to either turn on `thisVariantHasChanged` or do nothing at all, and whether we break early upon turning it on, or don't, it's the same outcome

also, it'd be nice to describe what bug this is fixing in the description, i'm not sure i know, and do you think we can test this so that we don't regress? i've added tests to this a while ago, they suck in that they're integration-y but it's better than the nothing we had before.. i'm fine with adding tests later if you can describe the problem and fix more clearly